omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -1186,23 +1213,50 @@
                 setValue('other-meds-quantity.error','');
             </restrictByRole>
 
-
                 var encounterDate = new Date((getValue('encounterDate.value')));
                 var appointment = new Date((jq("#returndate").find('input[type=hidden]').val()));
-                var transferOutDate = new Date((getValue('transfer_out_date.value')));
+                var transferOutDate = new Date((getValue('transfer_out_date.value'))); 
```

Comment:
@semujju I see extra parentheses in here

Reply:
@ssmusoke Let me clean this out